build.cmd

```diff
@@ -6,15 +6,24 @@ setlocal
 ::       means that that rebuilding cannot successfully delete the task
 ::       assembly. 
 
-:: Check prerequisites
-if not defined VS120COMNTOOLS (
-    if not defined VS140COMNTOOLS (
-        echo Error: build.cmd should be run from a Visual Studio 2013 or 2015 Command Prompt.  
-        echo        Please see https://github.com/dotnet/corefx/wiki/Developer-Guide for build instructions.
-        exit /b 1
+if not defined VisualStudioVersion (
+    if defined VS140COMNTOOLS (
+        call "%VS140COMNTOOLS%\VsDevCmd.bat"
+        goto :EnvSet
     )
+
+    if defined VS120COMNTOOLS (
+        call "%VS120COMNTOOLS%\VsDevCmd.bat"
+        goto :EnvSet
+    )
+
+    echo Error: build.cmd should be run from a Visual Studio 2013 or 2015 Command Prompt.  
```

Comment:
Sure, I can fix that.  Note that if you are already in a VS command prompt, we will respect it (which will allow building using the Dev12 tools even if you have Dev14 installed as well).